source/types/options.ts

```diff
@@ -177,7 +178,7 @@ export type KyOptions = {
 	const json = await ky('https://example.com', {fetch}).json();
 	```
 	*/
-	fetch?: (input: RequestInfo, init?: RequestInit) => Promise<Response>;
```

Comment:
This is breaking, but actually both [`@types/node`](https://github.com/nodejs/undici/blob/91ca13ccd065d50c47279e9fdac0c152464baee9/types/fetch.d.ts#L12) and [`lib.dom.d.ts`](https://github.com/microsoft/TypeScript/blob/ca7a3af5e607a6c8589f5f600d6c4960a8bc07d4/src/lib/dom.generated.d.ts#L25921) define it this way.